package.json

```diff
@@ -606,6 +611,10 @@
 			  "command": "bitbake.examine-dependency-taskexp",
 			  "group": "1@bitbake_dev@5"
 			},
+			{
+				"command": "bitbake.scan-global-env",
```

Comment:
Ideally we would not show this when right clicking recipes, but only targeted files like .conf and .bbclass.
The easy way to implement it in my opinion is keep the old button (scan recipe), but tweak the command to detect that when a .bbclass/.conf is passed, it should instead run the global parsing.

This would simplify the interface and keep an intuitive workflow.

What do you think?

Reply:
Your latest commit properly shows the correct command, however the new global scan command appears at the bottom of the menu, compared to the recipe scan command which is first of the second menu. Could you edit the order in package.json to use the same location as recipe scanning?

Reply:
This has not been addressed, you simply need to reorder and change the @ number in package.json so that the global environment scan button appears at the same place as the recipe scan button.

Reply:
Done